server/datastore/mysql/labels.go

```diff
@@ -226,7 +226,27 @@ func (d *Datastore) SaveLabel(ctx context.Context, label *fleet.Label) (*fleet.L
 
 // DeleteLabel deletes a fleet.Label
 func (d *Datastore) DeleteLabel(ctx context.Context, name string) error {
-	return d.deleteEntityByName(ctx, labelsTable, name)
+	return d.withRetryTxx(ctx, func(tx sqlx.ExtContext) error {
+		var labelID uint
+		row := tx.QueryRowxContext(ctx, `select id FROM labels WHERE name = ?`, name)
```

Comment:
Curious about the change from `sqlx.GetContext` to `tx.QueryRowxContext`.

Reply:
Because I keep forgetting that Get is part of sqlx and not part of tx... going back.

Reply:
Nit: On the schedule query deletion, we use the approach of `DELETE FROM` and then check for rows affected.